nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -403,3 +363,59 @@ def convert_to_config_list(initial_list):
     if type(initial_list[0]) is not ListConfig:
         initial_list = ListConfig([initial_list])
     return initial_list
+
+
+def get_chain_dataset(datasets, ds_config):
+    if len(datasets) > 1:
+        if ds_config.get('bucketing_batch_size', None) is not None:
+            bucketing_batch_sizes = calc_bucketing_batch_sizes(ds_config, len(datasets))
+            logging.info(
+                f"Batch bucketing is enabled for {len(datasets)} buckets with adaptive batch sizes of {bucketing_batch_sizes}!"
+            )
+            for idx, dataset in enumerate(datasets):
+                datasets[idx] = audio_to_text.BucketingDataset(
+                    dataset=dataset, bucketing_batch_size=bucketing_batch_sizes[idx]
+                )
+        else:
+            logging.info(
+                f"Batch bucketing is enabled for {len(datasets)} buckets with fixed batch size of {ds_config['batch_size']}!"
+            )
+
+    if len(datasets) == 1:
+        return datasets[0]
+    bucketing_strategy = ds_config.get('bucketing_strategy', 'synced_randomized')
```

Comment:
Let's add this config param to all of the major ASR model configs so it's visible. QuartzNet, Citrinet, ContextNet, Conformer (CTC and Transducer)

Reply:
Added.